app/services/avatar_generator.rb

```diff
@@ -1,12 +1,12 @@
 class AvatarGenerator
-  URL = 'https://api.adorable.io/avatars/80/'
-  FILE_FORMAT = '.png'
+  BASE_URL = 'https://api.adorable.io/avatars/'
+  IMAGE_SIZE = '80'
```

Comment:
Spoko 👍 Jeszcze bym zapisał 80 jako liczbę, a nie stringa, ponieważ tym de facto jest. Konwersja liczby na stringa to powinna być odpowiedzialność metody `generate_url`.